shared_model/cryptography/ed25519_sha3_impl/internal/CMakeLists.txt

```diff
@@ -15,22 +15,21 @@
 # limitations under the License.
 #
 
-add_subdirectory(impl)
-
 add_library(hash
         sha3_hash.cpp
         )
 
-target_link_libraries(hash
+target_link_libraries(hash PUBLIC
```

Comment:
PUBLIC is default value. https://cmake.org/cmake/help/v3.5/command/target_link_libraries.html
>target_link_libraries(<target> <item>...)
Library dependencies are transitive by default with this signature.